src/ZODB/tests/blob_transaction.txt

```diff
@@ -402,14 +398,7 @@ Now, the serial for the existing blob should be the same:
     >>> blob_storage.load(blob._p_oid, '') == (olddata, oldserial)
     True
 
-And we shouldn't be able to read the data that we saved:
-
-    >>> blob_storage.loadBlob(blob._p_oid, s1)
-    Traceback (most recent call last):
-    ...
-    POSKeyError: 'No blob file at <BLOB STORAGE PATH>'
-
-Of course the old data should be unaffected:
```

Comment:
Oh gawd, the machinery that loads blobs doesn't mind being called with None for the tid, so the last lines I removed above are meaningless. If you want to add a tid hack, be my guest, but I don't think it's necessary. I think the check:

     >>> blob_storage.load(blob._p_oid, '') == (olddata, oldserial)
     True

is sufficient.